subsys/bluetooth/controller/Kconfig.ll_sw_split

```diff
@@ -560,6 +560,15 @@ config BT_CTLR_ZLI
 	  shall not use Zero Latency IRQ themselves when this option is selected,
 	  else will impact controller stability.
 
+config BT_CTLR_DYNAMIC_INTERRUPTS
+	bool "Use Dynamic Interrupts"
+	depends on DYNAMIC_INTERRUPTS
+	select SHARED_INTERRUPTS
```

Comment:
Do we really need shared interrupts here? It would bloat the data size used, reserving another [number_of_interrupts] * [supported_number_of_shared_handlers] * (sizeof(func*) + sizeof(void*)) in the DATA space.

Reply:
Removed the selection of SHARED_INTERRUPTS